examples/mlflow_artifacts/docker-compose.yml

```diff
@@ -1,6 +1,9 @@
 version: "3"
 services:
   minio:
+    # Releases for 10/30/23 and 11/1/23 are broken.
+    # Once stability is returned to minio, unpin this version and revert to:
+    # image: minio/minio
```

Comment:
Found a solution: https://github.com/minio/minio/issues/18373#issuecomment-1790003599. Let's remove this comment.